### ISSUE TITLE
Feature Request: Option to redirect 404s to index.html/project

### ISSUE THREAD

**Issue by hgzimmerman** [koute/cargo-web#100]
Hello, this request pertains to Yew, and its future use of routing.
When a router is added to yew, it would be nice to have cargo-web's embedded webserver serve the app regardless of what the route in the browser url bar is.

In the course of using an app with routing, the url, which by default is `http://[::1]:8000`, might become `http://[::1]:8000/some/app/specific/route`. Understandably, if you refresh the page, the server will reply with a 404 message because it can't find a file at that resource. If an option to cause these 404s to just return the index.html so that a developer wouldn't have to navigate through their app to get to the prior route.

I think `cargo web start --return-index-on-404` or something like that would be a useful option to support.

**Comment by koute** [comment]
I don't see why not, although it would probably be better to have this as a `Web.toml` attribute?

**Comment by hgzimmerman** [comment]
Yes, I agree that it makes more sense to have this option in a config file rather than specifying it on the command line.
